scripts/install_evmone.ps1

```diff
@@ -5,5 +5,5 @@ $progressPreference = "silentlyContinue"
 
 Invoke-WebRequest -URI "https://github.com/ethereum/evmone/releases/download/v0.11.0/evmone-0.11.0-windows-amd64.zip" -OutFile "evmone.zip"
 tar -xf evmone.zip "bin/evmone.dll"
-mkdir deps
+mkdir deps -Force
 mv bin/evmone.dll deps
```

Comment:
What does `-Force` do here? Does it remove the submodules?

Reply:
Or does it just not fail if the directory already exists? We can probably remove this altogether can't we (the directory will always be there now)